root/etc/e-smith/events/actions/nethserver-nextcloud-conf

```diff
@@ -11,10 +11,16 @@ function OCC
 
 exitOnError () {
     echo $1
+    OCC "app:enable accessibility"
     OCC "maintenance:mode --off"
     exit 1
 }
 
+databaseTest () {
+    # if the database is not initialized then databaseSCL == 0 else > 0
+    databaseSCL=`mysql --socket=/run/rh-mariadb105-mariadb/nextcloud-mysql.sock -e "select count(*) from information_schema.tables where table_type = 'BASE TABLE' and table_schema = 'nextcloud'" | tail -n1`
```

Comment:
You could also avoid the `tail -n1` invocation by using:
```
 mysql --socket=/run/rh-mariadb105-mariadb/nextcloud-mysql.sock -BN -e "select count(*) from information_schema.tables where table_type = 'BASE TABLE' and table_schema = 'nextcloud'"
```

Reply:
great nice thank